features/page_objects/back_office/new/finance_payment_method_page.rb

```diff
@@ -12,7 +12,7 @@ class FinancePaymentMethodPage < SitePrism::Page
   element(:transfer, "#payment_form_payment_type_bank_transfer", visible: false)
   element(:missed_worldpay, "#payment_form_payment_type_worldpay_missed", visible: false)
 
-  element(:submit_button, ".button")
+  element(:submit_button, "input[value='Add payment details']")
```

Comment:
Can we use:
```suggestion
  element(:submit_button, "input[type='submit']")
```
If that works, I think is clearer and less open to update in case the text changes. WDYT?

Reply:
I'm happy with this. I had changed the submit button is so that it would be unique to that page and unambiguous for Capybara to detect it. But it didn't fix the non-headless issue I was having there, so I'm happy to switch to this. I checked that a test calling this still works (headlessly at least!)

Committed and pushed :)